### PR TITLE
Release v1.7.1

### PR DESCRIPTION
**Pull request by avalentino**
- [x] update the changelog
- [x] check auto-release feature
- [x] update version number an release date
- [ ] (re-)tag the new release
- [ ] upload source package and wheels
- [ ] open an issue to update the conda-forge pyerfa-feedstock for conda packages

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
 
 - Bundled liberfa version update to v1.7.1.
+- Now it is possible to build against system liberfa [gh-39].
 - Improved the setup machinery to ensure a proper configuration of the
   embedded liberfa [gh-53] (see also https://github.com/liberfa/erfa/pull/73).
 - Improve version testing [gh-53] (see issue gh-52 for rationale).
+- Reworked version management [gh-57].
+- Make pyerfa build reproducible [gh-46] (see issue gh-45 for rationale).
+- Improved docstring titles [gh-47].
+- Most of the CI has been moved to GutHub Actions.
```
Comment (mhvk) [inline CHANGES.rst:12]: Not sure this belongs in the user-facing `CHANGES.rst`, but if we keep it: GutHub-> GitHub

Comment (avalentino) [reply CHANGES.rst]: done

**Comment by mhvk** [comment]
Changelog changes look good. Do you want to try to proceed with adding a release date? Once we have merged the PR, all that should be needed for actually doing the release is for you to update master locally, to tag it with v1.7.1, and push the tag upstream - @astrofrog's magic setup will then automatically make a release for pypi (presuming the previous one failed because of broken version numbering; I deleted the existing tags already).

**Review by mhvk — APPROVED** [review]
Approving so you can proceed...

**Comment by avalentino** [comment]
Thanks @mhvk

**Pull request merged**